marimo/_output/md.py

```diff
@@ -215,3 +218,45 @@ def md(text: str) -> Html:
     - An `Html` object.
     """
     return _md(text)
+
+
+def latex(*, filename: Union[str, Path]) -> None:
+    """Load LaTeX from a file or URL.
+
+    ```python
+    import marimo as mo
+
+    mo.latex(filename="macros.tex")
+    ```
+
+    or
+
+    ```python
+    import marimo as mo
+
+    mo.latex(url="https://example.com/macros.tex")
```

Comment:
@mscolnick is this functionality in? Or did you mean `filename`?

Reply:
whoops, i did, thanks